### PR TITLE
Query PSD01 changes

### LINKED ISSUE #314: Wrong URI for the editors draft
Should be 

https://oslc-op.github.io/oslc-specs/specs/query/oslc-query.html

instead of 

https://tools.oasis-open.org/version-control/svn/oslc-core/trunk/specs/oslc-query.html

### PR DESCRIPTION
**Pull request by DavidJHoney**
Fixed #314
Query PSD01 changes

### CODE REVIEW

**Comment by DavidJHoney** [comment]
> 
> 
> The working group is still set as the OSLC TC - this should now be the OSLC Open Project (compare with Core), with Andrew as the co-chair in place of Martin.

Fixed that with new commit+push. Incremented release date. Created new zip and attached to release.

**Comment by berezovskyi** [comment]
@DavidJHoney I don't see your commit in this branch. Are you sure you committed to this branch and pushed?

File: specs/query/oslc-query.html [hunk 1]
```diff
@@ -96,13 +96,23 @@
...
-        // name of the TC
-        wg: "OASIS OSLC Lifecycle Integration Core (OSLC Core) TC",
-        wgShortName: "oslc-core",
-
-        // URI of the public TC page
-        wgURI: "https://www.oasis-open.org/committees/oslc-core/",
+        
+        // =====  Overall OSLC OP settings =====
+
+        wg: "OASIS OSLC OP",
```
Comment (berezovskyi) [inline specs/query/oslc-query.html:102]: I think it should be "OASIS Open Services for Lifecycle Collaboration (OSLC) OP", just checked other published PSDs.

**Comment by berezovskyi** [comment]
@jamsden could you please coordinate PSD spec publication with @chet-ensign?

**Pull request merged**